packages/rich-text/src/test/change-list-type.js

```diff
@@ -17,7 +17,8 @@ describe( 'changeListType', () => {
 
 	it( 'should only change list type if list item is indented', () => {
 		const record = {
-			formats: [ , ],
+			formats: [],
```

Comment:
Is this meant to have changed, or should it have remained ?

```suggestion
			formats: [ , ],
```

A few other occurrences in the file. Unsure if it's the same sort of "the implementation doesn't bother with it, so its value doesn't matter" as in my previous comment.

Reply:
Thanks! Removed in 8c9def8.